pyproject.toml

```diff
@@ -69,16 +69,18 @@ types-python-dateutil = { version = "*" }
 requests_auth = { version = "*" }
 keyring = { version = "*" }
 
-[tool.poetry.group.docs]
+[tool.poetry.group.doc]
 optional = true
 
-[tool.poetry.group.docs.dependencies]
+[tool.poetry.group.doc.dependencies]
 # Doc packages
 ansys-sphinx-theme = { version = "0.15.2" }
 numpydoc = { version = "1.7.0" }
 sphinx = { version = "7.3.7" }
 sphinx-notfound-page = { version = "1.0.0" }
 sphinx-copybutton = { version = "0.5.2" }
+# Duplicate extra - required to use ansys/actions/doc-build
+requests-kerberos = {version = "^0.13", markers = "sys_platform == 'linux'" }
```

Comment:
We ought to submit a ticket to get extras supported by actions

Reply:
I ended up extracting a dependency to a new group. 

Dev dependencies are not optional, so they get installed by `poetry install`, which is what actions typically use. Because `asgi_gssapi` requires `gssapi`, most jobs required installing kerberos headers.
It's now installed via the tox env config, and thus is only required for tests.